Sources/EthWrapper/ResolutionConfig.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  File.swift
+//  
+//
+//  Created by Stephen Chase on 22/03/2023.
+//
+
+import Foundation
+
+public struct ResolutionConfig: Decodable {
+    public let erc20Tokens: Bool
+    public let nfts: Bool
```

Comment:
please tripple check I think the plurals are not correct.

Reply:
I'm seeing this:
{ erc20: true, externalPlugins: true, nft: true}

Reply:
Will do, might have referenced the wrong func